# origin coordinates of the object
float64 origin_x
float64 origin_y
float64 origin_z

# dimensions of the object
float64 dimension_x
float64 dimension_y
float64 dimension_z

# destination coordinates of the object
float64 destination_x
float64 destination_y
float64 destination_z
